### PR TITLE
feat(manager): SPAship dashboard revamp with the new design

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

Implemented design revamp for 
1) Login Screen
2) Dashboard at MAin level
3) Dashboard at SPA and Web property level

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/views/DashboardPage/components/ActivityStreamDashboard/ActivityStreamDashboard.tsx [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-    >
-      <SimpleBarReact style={{ maxHeight: 830, padding: '10px' }}>
-        <div style={{ marginTop: '20px', marginLeft: '10px' }}>
-          <ActivityStream action="APPLICATION_DEPLOYED" isGlobal />
-        </div>
-      </SimpleBarReact>
-    </Card>
-  </>
+  <Card
+    // isSelectable
```
Comment (soumyadip007) [inline packages/manager/src/views/DashboardPage/components/ActivityStreamDashboard/ActivityStreamDashboard.tsx:9]: remove this line

File: packages/manager/src/views/LoginPage/components/Nav/Nav.tsx [hunk 2]
```diff
@@ -29,16 +29,31 @@ export const Nav = () => (
...
+              <img src="/img/slack-logo.svg" alt="slack logo" />
+              <Button
+                className="pf-u-pl-sm"
+                component="a"
+                aria-label="Documentation"
+                variant={ButtonVariant.link}
+                rel="noopener noreferrer"
               >
-                Documentation
+                Spaship Forum
```
Comment (soumyadip007) [inline packages/manager/src/views/LoginPage/components/Nav/Nav.tsx:56]: make it SPAship

**Review by soumyadip007 — COMMENTED** [review]
@nikhitamore please resolve the comments

File: packages/manager/src/views/DashboardPage/components/ActivityStreamDashboard/ActivityStreamDashboard.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/views/DashboardPage/components/ActivityStreamDashboard/ActivityStreamDashboard.tsx]: done

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Pull request merged**